### PR TITLE
[#100906588] udpate CHANGELOG

### PR DESCRIPTION
**Pull request by marexandre**

### CODE REVIEW

**Comment by yugui** [comment]
LGTM

File: CHANGELOG.rst [hunk 1]
```diff
@@ -3,6 +3,12 @@ Changelog
 
 Only important changes are mentioned below. See `commit log <https://github.com/gengo/gengo-python/commits/master>`_ and `closed issues <https://github.com/gengo/gengo-python/issues?state=closed>`_ for full changes.
 
+v0.1.28 (2015-10-07)
+-------------------
+
+* [Feature] `#59 <https://github.com/gengo/gengo-python/pull/59`_ Add attachments support for Order and Job comments
```
Comment (narusemotoki) [inline CHANGELOG.rst]: `59` -> `59>`

**Comment by narusemotoki** [comment]
LGTM

File: CHANGELOG.rst (hunk 1 above)
Comment (marexandre) [inline CHANGELOG.rst]: :bow:

**Comment by hayate** [comment]
@marexandre thank you LGTM

**Pull request merged**